_data/projects/eui-event-template.yml

```
@@ -0,0 +1,10 @@
+name: React and Eui Event Template
+desc: A React app to share information about an event or conference, such as speaker details, agenda, and event actions
+site: https://eui-event.vercel.app/
+tags:
+  - javascript
+  - reactjs
+  - beginner
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/brittanyjoiner15/eui-event-template
```

🎨 this should be a deep link to the issues tagged with the `up for grabs` label:

```suggestion
  link: https://github.com/brittanyjoiner15/eui-event-template/labels/up%20for%20grabs
```